src/builder.ts

```diff
@@ -102,8 +102,13 @@ export function generateManifest(
   ajv.addSchema(publicationSchemas);
   const valid = ajv.validate(publicationSchemaId, publication);
   if (!valid) {
+    const errors = ajv.errors
+      ?.map((value, index, array) => {
```

Comment:
`index` `array` は使用していないため、省略可能です。

```suggestion
      ?.map((value) => {
```